tests/testthat/test_getobservation.R

```diff
@@ -624,7 +624,7 @@ test_that("POX with offering object in list and not in list (2.0.0)", {
 # FOI retrieval can be disabled (tested with POX, 2.0.0) ----
 #
 test_that("FOI retrieval can be disabled (tested with POX, 2.0.0)", {
-  #skip_on_cran()
+  skip_on_cran()
```

Comment:
I don't think that unskipping this test solves a problem on CRAN. I suggest to skip more, not less tests :-)

Reply:
that's what we do

Reply:
Ahh, sorry. I should practice reading diffs..